openfe/protocols/openmm_rfe/equil_rfe_settings.py

```diff
@@ -519,19 +424,35 @@ class RelativeHybridTopologyProtocolSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
+    # Inherited things
+
+    forcefield_settings: OpenMMSystemGeneratorFFSettings
+    """Parameters to set up the force field with OpenMM Force Fields."""
+    thermo_settings: ThermoSettings
+    """Settings for thermodynamic parameters."""
+
     # Things for creating the systems
     system_settings: SystemSettings
+    """Simulation system settings including the long-range non-bonded method."""
     solvation_settings: SolvationSettings
+    """Settings for solvating the system."""
 
     # Alchemical settings
     alchemical_settings: AlchemicalSettings
+    """Alchemical protocol settings including lambda windows and soft cores."""
     alchemical_sampler_settings: AlchemicalSamplerSettings
+    """Settings for sampling within lambda windows."""
```

Comment:
```suggestion
    """
    Settings for alchemical space sampling, including the number of lambda windows
    and simultaneous sampling replicas.
    """
```
Something like that, simultaneous sampling replicas is quite a mouthful so feel free to reject. P.S. forgive the formatting it's late.

Reply:
The number of lambda windows is in `AlchemicalSettings`, and alchemical space sampling sounds to me like it would be searching through a space that includes different alchemical states. `AlchemicalSamplerSettings` seems focused on how to sample each window - could you clarify why my phrasing is incorrect/inadequate?

Reply:
re: number of lambda windows - you're 100% right, I'm muddling myself with n_replicas.

re: why I think we need to maybe reword - beyond "sampling within lambda space", these settings also tackle things like the online analysis interval, and (unfortunately) the number of whole repeats of the entire simulation. I was trying (and probably failing to) reword it as something that covers that wider space.

Reply:
Gotcha, I misunderstood how the protocol worked.